src/constants.ts

```diff
@@ -144,3 +144,8 @@ export const SearchTypeValues = [
   'concept',
   'podcast-series',
 ] as const;
+
+export enum Revision {
+  REVISED = 'revised',
+  NEEDSREVISION = 'needs-revision',
+}
```

Comment:
Vi bruker generelt sett ikke enums i kodebasen. Det er et par drawbacks med de som vi kan unngå ved å bruke string unions. Pitcher heller at vi gjør det!

Reply:
Nå er jeg ikke like bekymra for drawbacks ved enums så lenge de brukes riktig. Men innretter meg :)